src/resolvers.ts

```diff
@@ -26,6 +26,46 @@ export const resolvers = {
         birthDate: user.birthDate,
         cpf: user.cpf
       }
+    },
+
+    users: async (parent, args, context) => {
+      verifyToken(context.request.headers.authorization);
+      let quantity = 5;
+      if (args.quantity) {
+        quantity = args.quantity;
+      }
+
+      let skip = 0;
+      if (args.skip) {
+        skip = args.skip;
+      }
+
+      let before: boolean = true;
+      if (skip == 0) {
+        before = false;
+      }
+
+      const userRepository = getRepository(User);
+
+      const userCount = await userRepository.count();
+      let after: boolean = false;
+      if (userCount > quantity + skip) {
+        after = true;
+      }
+
+      const users = await getConnection()
+      .getRepository(User)
+      .createQueryBuilder("user")
+      .orderBy("user.name", "DESC")
+      .take(quantity)
+      .skip(skip)
+      .getMany();
```

Comment:
```suggestion
      const users = await getConnection()
      .getRepository(User)
      .createQueryBuilder("user")
      .orderBy("user.name", "DESC")
      .take(quantity)
      .skip(skip)
      .getMany();
```
```suggestion
      const users = await getConnection()
        .getRepository(User)
        .createQueryBuilder("user")
        .orderBy("user.name", "DESC")
        .take(quantity)
        .skip(skip)
        .getMany();
```
A indentação tá errada aqui (e alguns outros lugares). Depois de terminar as atividades ou quando tiver um tempinho sobrando, dá uma olhada no prettier